numba/tests/test_array_methods.py

```diff
@@ -894,6 +900,27 @@ def test_imag(self):
         z = x + 1j*y
         np.testing.assert_equal(pyfunc(z), cfunc(z))
 
+    def test_clip(self):
+        a = np.linspace(-10, 10, 101)
+        pyout = np.empty_like(a)
+        cout = np.empty_like(a)
```

Comment:
Note to self: these should be moved inside of the for loop, to prevent cross-contamination from the last test (`np.testing.assert_equal(pyout, cout)`).